workspaces/liferay-partner-workspace/client-extensions/liferay-partner-custom-element/src/common/interfaces/tableColumn.ts

```diff
@@ -6,5 +6,6 @@
 export default interface TableColumn<T> {
 	columnKey: string;
 	label: string | JSX.Element;
+	label2?: string;
```

Comment:
The label prop accept JSX.Element, you should use this to create a dynamic label.

```suggestion
	label: string | JSX.Element;
```